### ISSUE TITLE
reviewer 1's comments

### ISSUE THREAD

**Issue by qntkhvn** [getstrained/intro-strain#11]
**Model assumptions**

- [x]  I really liked the multi-level model for STRAIN. That said, the model encodes some strong exchangeability assumptions that, in my opinion, are worth probing. Insofar as (i) pass blockers are nested within teams and (ii) different teams deploy different pass protection schemes/techniques, have you considered an elaborated model in which pass blocker effects are first shrunk towards a team-specific average, which themselves are shrunk to a league-wide average? Perhaps more importantly, as you noted in Section 4.3, the model ignores correlated performance between defensive linemen on the same team. To account for this, you could, similarly nest pass rushers within teams. Doing so, however, introduces positive dependence between the $R_j$'s and it is not obvious to me that two teammates' strains on the same play should be positively correlated. We might observe negative dependence, for instance, if pass blockers concentrated more on blocking one player over another. At the very least, you should discuss the assumptions underpinning the multi-level model. A clear demonstrate that the proposed model provides a better fit to the data than alternatives would, of course, be much better!

We thank the reviewer for the positive feedback and for their feedback about the multi-level model. We agree with the reviewer's concern and have updated the model to account for both pass rush and blocking team effects, with nested players. We demonstrate in our supplementary material that the team model with nested player effects performs better out-of-sample in comparison to our previous model specification.  Coupled with this change, in order to provide uncertainty about the new combined team+player effects, we switched from a parametric bootstrap to a resampling based approach which we believe provides more insight about the model estimates without the rigid assumptions of parametric bootstrapping. We recognize limitations of our initial modeling efforts of STRAIN, and have included further discussion of a RAPM-style approach to address these concerns in Section 5.

- [x] Relatedly, is there a reason to model the average STRAIN across a play as opposed to the frame-level STRAIN values? The two example plays reveal considerable variation in the frame-to-frame variability. I worry that simply averaging over all frames in a play might obscure this variation.

This is a great point raised by the reviewer. Our STRAIN curve figures clearly demonstrate frame-level variability that averaging may hide. We have updated the final paragraph of Section 5 to emphasize this point more clearly. However, we leave modeling frame-level STRAIN for future work due to the increased complexity of that problem and its necessary techniques in relation to this manuscript's purpose of introducing STRAIN in American football. 

**Strain curves**

- [x] I was a bit confused by the STRAIN curves in Figure 7. Are these curves based on a single play? Are they averages of the observed $\overline{\text{STRAIN}}_{ij}$'s values across all plays and all players within positions? Or are they averages of the estimated $\overline{\text{STRAIN}}$ values from the fitted multi-level model? If these curves are based solely on the observed $\overline{\text{STRAIN}}$ values, to what extent can the positional differences be explained by the fact that different pass rush positions tend to be blocked by different offensive line positions? Since multi-level model allows you to adjusting for the position of the nearest pass blocker, it would be interesting to see whether the differences observed between the curves persist after such adjustment.

The STRAIN curves are not based on the multilevel and instead are based on the observed average STRAIN values for each position. We have added a sentence to clarify this at the start of section 4.2. It now says "For each position, the curve is based on the average of the observed STRAIN at each frame across all plays and players within the position."

**Minor comments**

- [x] The $f_{d_{ij}}(t)$ notation in Equations (3)–(5) seems cumbersome. What is $f$ and why is indexed by a distance? Why not just use $d_{ij}(t)$ for the distance?

We have changed the distance notation from $f_{d_{ij}}(t)$ to $s_{ij}(t)$. We chose $s_{ij}(t)$ instead of $d_{ij}(t)$ (as suggested by the reviewer) since $s_{ij}(t)$ aligns with the common notation for distance/position in physics. Further, we wanted to avoid confusion since we already had another $d$ notation for differentiation.

- [x] Fixed effects in the multi-level model: it would be helpful to describe what variables are recorded in $\mathbf{x}$ when you introduce the multi-level model instead of in a table towards the end.

We have added more descriptions of the fixed effects in our multilevel model. See the last paragraph of Section 3.2 in our revised paper.

**Comment by qntkhvn** [comment]
@gjm112 to clarify STRAIN curves description in the text (i.e. where we refer to Figure 7)

**Comment by qntkhvn** [comment]
@ryurko to fix the last minor comment

**Comment by gjm112** [comment]
How is position defined in this data?  Is it done by PFF?  Or is it done by the NFL?

**Comment by qntkhvn** [comment]
"official position" by NFL - PFF only gives player "roles"

**Comment by qntkhvn** [comment]
close in favor of #18
